spec/models/concerns/digital_object_concerns/index_behavior_spec.rb

```diff
@@ -20,7 +20,6 @@
 
   context "#destroy" do
     it "runs as expected, removing the destroyed object from the index" do
-      digital_object_with_sample_data.save
```

Comment:
Test seems fine without this line (since the object is added to the index when `FactoryBot.create` runs earlier, but @barmintor please let me know if it's a mistake for me to remove it from your test.

Reply:
I wonder if that's an artifact from using `new` instead of `create` at some point

Reply:
Ah yeah, that could be.  Are you okay with my removal then?

Reply:
Yeah, if the test is running it's unnecessary.